3.py

```diff
@@ -0,0 +1,4 @@
+a = []
+for i in range(3):
```

Comment:
давай уже
```suggestion
a = list(map(lambda x: int(input()), range(3)))
```
чего уж там))